tests/suites/test_suite_bignum.function

```diff
@@ -905,6 +905,76 @@ exit:
 }
 /* END_CASE */
 
+/* BEGIN_CASE */
+void mpi_div_mpi_aliasing( int result )
```

Comment:
Same as https://github.com/Mbed-TLS/mbedtls/pull/6588/files#r1021216383: please don't introduce this new test structure, the established one is better.